pkg/cvo/metrics.go

```diff
@@ -83,6 +86,14 @@ version for 'cluster', or empty for 'initial'.
 			Name: "cluster_installer",
 			Help: "Reports info about the installation process and, if applicable, the install tool.",
 		}, []string{"type", "version", "invoker"}),
+		clusterVersionOperatorUpdateRetrievalTimestampSeconds: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cluster_version_operator_update_retrieval_timestamp_seconds",
+			Help: "Reports when updates were last succesfully retrieved.",
+		}, []string{"name"}),
+		clusterVersionOperatorReasonFailure: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cluster_version_operator_reason_failure",
```

Comment:
Both `clusterVersionOperatorReasonFailure` and `cluster_version_operator_reason_failure` are too generic for failures to fetch *update* retrievals.  How about `cluster_version_operator_update_retrieval_failure` or some such?  And... do we want to record a non-empty reason on success too (like `AsExpected`?).  Maybe this should be `cluster_version_operator_update_retrieval_conditions` (like the existing `cluster_operator_conditions`)?